installer/install.iss

```diff
@@ -1524,6 +1564,19 @@ begin
 
     PrevPageID:=wpSelectProgramGroup;
 
+    (*
+     * See if this is an upgrade, and if so, create a custom page to allow the user to select whether or not to do a fast install.
+     *)
+    CurrentVersion:=ExpandConstant('{#APP_VERSION}');
+    if ISUpgrade(CurrentVersion,PreviousGitForWindowsVersion) then begin
```

Comment:
Please use the same case as in the function declaration: `IsUpgrade` (the `s` should be lower-case).